waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -97,3 +111,9 @@ getApp spec = case takeDecls @App (AS.decls spec) of
 -- | This function assumes that @AppSpec@ it operates on was validated beforehand (with @validateAppSpec@ function).
 isAuthEnabled :: AppSpec -> Bool
 isAuthEnabled spec = isJust (App.auth $ snd $ getApp spec)
+
+isPostgresUsed :: AppSpec -> Bool
+isPostgresUsed spec = Just AS.Db.PostgreSQL == (AS.Db.system =<< AS.App.db (snd $ getApp spec))
+
+isPgBossJobExecutorUsed :: AppSpec -> Bool
+isPgBossJobExecutorUsed spec = any (\(_, job) -> Job.executor job == Job.PgBoss) (AS.getJobs spec)
```

Comment:
True. Trick is I can't put in `src/Wasp/AppSpec/Job.hs` where I'd like to, since `AppSpec` imports it, so it would create a cycle. I can perhaps create a `src/Wasp/AppSpec/Util.hs`

Reply:
Yup, that sounds like a good idea!